### ISSUE TITLE
Inconsistent cost predictions vs transactions vs renter info

### ISSUE THREAD

**Issue by TacoByte** [NebulousLabs/Sia-UI#328]
Darwin 1.0

I bought 10GB of storage. It was quoted at around 1500 SC.

My wallet then sent many transactions, with a total well in excess of the original quote, around 10000 SC.  This [screenshot](http://i.imgur.com/ouorPvo.jpg) shows most of the transactions that were sent in this batch.

This is what renter returns, which seems inconsistent as well (the 5346 transaction alone seems larger than the allocated funds)

```
Renter info:
    Storage Spending:  25.43 mS
    Upload Spending:   20.97 SC
    Download Spending: 0 H
    Unspent Funds:     3.932 KS
    Total Allocated:   3.953 KS

Tracking 1 files:
 252.0 KB  1453756944023.png (uploading, 8.33%)
```

I don't know if this is "just" the UI misreporting things, or if something more lower level is going on.

**Comment by DavidVorick** [comment]
The renter is saying that it only used 4,000 coins between all of the contracts. Not sure why that's such a gap from 1500.

The wallet is reported that 10,000 coins have been spent?

**Comment by TacoByte** [comment]
If I add up all the transactions that happened when I bought the 100GB, I get around 7000 SC of spending (not quite 10000, sorry for rounding up so aggressively :) but still far away from 1500)

**Comment by DavidVorick** [comment]
Ok. That's definitely not right, I have some ideas about why though.

**Comment by DavidVorick** [comment]
The contract formation right now is failing with a large percentage of the hosts, which means that the price estimation is off. The price estimation and formation use the cheapest hosts, but if too many cheap hosts fail, you will switch to the more expensive hosts, resulting in a bad estimation.

**Comment by TacoByte** [comment]
I was actually suspecting something like that. The host market is so illiquid right now that it results in a lot of granularity and discrepancies between what Sia predicts will happen based on available information and what deals actually do get passed on the market/blockchain.

**Comment by TacoByte** [comment]
That still doesn't explain why renter reports spending 4 KS when my outgoing transactions add up to more than that?

Also, I uploaded some more files (well within my already concluded 10GB contract), and more SC is being sent from the wallet? -584.58 I was expecting to not have to spend any more once I had signed the contracts?

**Comment by superdcc** [comment]
Same Problem
10 GB, Estimated 1300,But really cost 7500 
And still show : not enough contracts to upload files  (when upload file)

[ScreenShot](http://imgur.com/JS3Vx4A)

**Comment by avahowell** [comment]
fixed in #409

**Comment by DavidVorick** [comment]
To clarify, there were some changes on the front end as well as the back end, multiple bugs were causing such heavy overspending, and now most are fixed. Estimates will still be off sometimes, but usually by less than 20% instead of being 1500 vs 7500.

More improvements are being pursued.
